opencivicdata/elections/models/election.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Election-related models.
+"""
+from __future__ import unicode_literals
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from opencivicdata.core.models.base import (
+    OCDBase,
+    IdentifierBase,
+    LinkBase,
+    OCDIDField,
+)
+from opencivicdata.core.models import (
+    Division,
+    Organization,
+)
+
+
+@python_2_unicode_compatible
+class Election(OCDBase):
+    """
+    A collection of political contests set to be decided on the same date within a Division.
+    """
+    id = OCDIDField(
+        ocd_type='election',
+        help_text='Open Civic Data-style id in the format ``ocd-election/{{uuid}}``.',
+    )
+    name = models.CharField(
+        max_length=300,
+        help_text='Name of the Election.',
+    )
+    date = models.DateField(
```

Comment:
IMO even vote by mail elections will want to be tied to a specific date on which the votes are expected to be counted and the results will be published. But, yeah, a convo for another time when we know users will need to track when ballots can be cast.